includes/general.admin.inc

```diff
@@ -24,6 +24,13 @@ function islandora_solr_metadata_general_admin_form($form, &$form_state) {
       '#description' => t('Only show unique values from each field. NOTE: Uniqueness is checked on values directly out of Solr. Later formatting of the value could break uniqueness.'),
       '#default_value' => variable_get('islandora_solr_metadata_dedup_values', FALSE),
     ),
+    'islandora_solr_metadata_field_value_separator' => array(
+      '#type' => 'textfield',
+      '#title' => t('Field value separator'),
+      '#description' => t('Characters to separate values in multivalued fields.'),
+      '#default_value' => variable_get('islandora_solr_metadata_field_value_separator', "\n"),
```

Comment:
I don't think this whole \n business is really gonna work ...

- It's a programming convention and not really UI-friendly
- It's not gonna show up in the config
- It can't easily be entered in the config (typing \n in there doesn't really give you what you want either)
- Saving the config with it in there as the default value is considered 'invalid' and makes it impossible to save the form, even though what's in there is technically valid

A possible compromise here would be to not have the field be required, and allow it to be left blank to use newlines as separators.